addon/initializers/component-styles.js

```diff
@@ -0,0 +1,36 @@
+import Ember from 'ember';
+import podNames from 'ember-component-css/pod-names';
+
+const {
+  Component,
+  ComponentLookup,
+} = Ember;
+
+export function initialize() {
```

Comment:
This work should not be done inside the initialize function (if it is it will be reran once for every test. It should be done in the root of the module so that it is only done once when the module is evaluated (and before the application itself is created).